shop.js

```diff
@@ -95,4 +97,27 @@ function showShoppingCartProducts() {
         totalPrice = totalPrice + shoppingCart[i].price;
     }
     document.getElementById('total').textContent = totalPrice.toFixed(2) + '€';
+};
+
+function getInvoice() {
+    const email = document.getElementById('emailInputField').value;
+
+    let body = 'Thank you very much for your registration at Krioukov Academy.<br>Here are the courses you are interested in: <ul>'
+    for (let i = 0; i < shoppingCart.length; i++) {
+        body = body + `<li>${shoppingCart[i].title} (${shoppingCart[i].dates}) - ${shoppingCart[i].price}€</li>`;
+    }
+
+    body = body + '</ul><br>We will contact you soon to clarify the last details. <br>Best regards, SBKG'
+    
+    Email.send({
+        SecureToken : "16f7332e-9824-4cbd-b4f0-7b9904b66af6",
```

Comment:
uh...yeah, that's true that you need to have a backend which would store your key(s).

In that case I would recommend to remove this dependency to your personal account and at least create a new one (test one for now). And then use secure key related to it. Because right now anyone who would get this token and your email can send any emails to anyone and it would looks like you did it. Which is quite dangerous.

Reply:
Hi! I removed the token for time. Thank you, I will try to create a test account not to use mine.